plenum/server/replica_validator.py

```diff
@@ -60,6 +56,14 @@ def validate_3pc_msg(self, msg):
         if view_no == self.replica.viewNo and node.view_change_in_progress:
             return STASH_VIEW, FUTURE_VIEW
 
+        # If Catchup in View Change finished then process a message
+        if node.is_synced and node.view_change_in_progress:
+            return PROCESS, None
+
+        # 6. Check if Participating
+        if not node.isParticipating:
```

Comment:
Please add the tests for conditions 5 and 6.

Reply:
Also we need an integration test that
1) starts a view change with on a Node, where last_ordered=(v, X), last_prepared_certificate is (v, X+5)
2) catch-up is started, and while doing catch-up, the node receives Commits for last_prepared_certificate.
Expected results: 
1) view change is finished successfully
2) view changed is finished after just 1 round of catch-up because last_prepared_cert is reached.